<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Minecraft Server</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtmcserver.qrc">
    <normaloff>:/images/qtmcserver-256x256.png</normaloff>:/images/qtmcserver-256x256.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabConsole">
       <attribute name="title">
        <string>Console</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QTextEdit" name="serverLogTextEdit">
          <property name="font">
           <font>
            <family>Lucida Console</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="sendCommandButton">
          <property name="text">
           <string>&amp;Send</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="serverCommandLineEdit">
          <property name="placeholderText">
           <string>Command</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabServerProperties">
       <attribute name="title">
        <string>Server Properties</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextEdit" name="serverPropertiesTextEdit">
          <property name="font">
           <font>
            <family>Lucida Console</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Server">
    <property name="title">
     <string>&amp;Server</string>
    </property>
    <widget class="QMenu" name="menu_Properties">
     <property name="title">
      <string>&amp;Properties</string>
     </property>
     <addaction name="actionSaveServerProperties"/>
     <addaction name="actionRefreshServerProperties"/>
    </widget>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="menu_Properties"/>
   </widget>
   <widget class="QMenu" name="menu_Log">
    <property name="title">
     <string>&amp;Log</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Server"/>
   <addaction name="menu_Log"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionRefreshServerProperties"/>
   <addaction name="actionSaveServerProperties"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit Qt Minecraft Server</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About Qt Minecraft Server</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="toolTip">
    <string>Qt Minecraft Server Settings</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start Minecraft Server</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Sto&amp;p</string>
   </property>
   <property name="toolTip">
    <string>Stop Minecraft Server</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear Console Log</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/exportlog.png</normaloff>:/images/exportlog.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export ...</string>
   </property>
  </action>
  <action name="actionSaveServerProperties">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save Server Properies</string>
   </property>
  </action>
  <action name="actionRefreshServerProperties">
   <property name="icon">
    <iconset resource="qtmcserver.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh Server Properties</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mainTabWidget</tabstop>
  <tabstop>serverLogTextEdit</tabstop>
  <tabstop>serverCommandLineEdit</tabstop>
  <tabstop>sendCommandButton</tabstop>
  <tabstop>serverPropertiesTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="qtmcserver.qrc"/>
 </resources>
 <connections/>
</ui>
